### ISSUE TITLE
mention build dependencies in the README

### ISSUE THREAD

**Issue by danieleades** [Hirevo/alexandrie#36]
Diesel has requirements on system packages, depending on which features are installed. These are the drivers for the different databases.

The fact that these dependencies must be installed (or you get linker errors during build) should be mentioned

for example, `libpq` (package `libpq-dev`) is a required system dependency when using the `diesel/postgres` feature, which is enabled by default

**Comment by Hirevo** [comment]
Yes, we're indeed have system dependencies that could be mentioned.  
I'll go compile the project with a bunch of different features to see which system dependencies end up being required.
